lua/nvim-tree/renderer/builder.lua

```diff
@@ -91,46 +97,48 @@ local function get_folder_name(node)
   return name
 end
 
-function Builder:_unwrap_git_data(git_icons_and_hl_groups, offset)
-  if not git_icons_and_hl_groups then
+---@class HighlightedString
+---@field str string
+---@field hl string|nil
```

Comment:
Thanks for starting this documentation pattern. We should use it more.

`sumneko_lua` gives me a warning however it does that for other lua code. I might look at putting a suppression in.

![20221230_171338 1052x253](https://user-images.githubusercontent.com/1505378/210040345-ac0b97d4-bf7b-4297-976a-c6feb0553583.png)

Reply:
hmmm interesting, it doesn't give me any warnings. But maybe putting in suppression in `.luarc.json` is a good idea for this one.

Reply:
The problem was on my end... I was putting the current directory in the workspace twice.

I've actually managed to remove all the suppressions: https://github.com/nvim-tree/nvim-tree.lua/commit/e322fbb80b0be5e885268c91ab29cbfe0143cd5b